daniel-alejandro-garcia/tarea-11/to-do-list-routing/src/index.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import App from './App';
+import reportWebVitals from './reportWebVitals';
+import  {BrowserRouter, Routes, Route} from 'react-router-dom';
+import TaskOne from './components/taskOne';
+import TaskTwo from './components/taskTwo';
+import TaskThree from './components/taskThree';
+import TaskFour from './components/taskFour';
+import TaskFive from './components/taskFive';
+import TaskSix from './components/taskSix';
+import TaskSeven from './components/taskSeven';
+import TaskEight from './components/taskEight';
+import TaskNine from './components/taskNine';
+import TaskTen from './components/taskTen';
+import TaskEleven from './components/taskEleven';
+
+ReactDOM.render(
+  <BrowserRouter>
+    <Routes>
+      <Route path='/' element={<App />}/>
```

Comment:
@da-garcia , this route list should be dynamic, and built from an array using map function